sdk/cpp/core/src/netconf_service.cpp

```diff
@@ -318,35 +399,20 @@ bool NetconfService::validate(NetconfServiceProvider& provider, DataStore source
 
 bool NetconfService::validate(NetconfServiceProvider& provider, Entity& source)
 {
-    YLOG_INFO("Executing validate RPC");
+    YLOG_INFO("Executing 'validate' RPC on [{}]", source.get_segment_path());
 
     // Get the root schema node
     shared_ptr<path::Rpc> rpc = get_rpc_instance(provider, "ietf-netconf:validate");
 
     // source
-    std::string entity_string = get_data_payload(source, provider.get_session().get_root_schema());
+    std::string entity_string = get_data_payload(source, provider);
     rpc->get_input_node().create_datanode("source/config", entity_string);
 
     auto read_datanode = (*rpc)(provider.get_session());
     return read_datanode == nullptr;
 }
 
-string to_string(DataStore datastore)
```

Comment:
This function actually is not a static function. It is declared in `types.hpp`. So, if deleting this function, please also delete the declaration from `types.hpp`